build/BUILD.bazel

```diff
@@ -15,25 +15,27 @@
 # JAX is Autograd and XLA
 
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
+load("@org_tensorflow//tensorflow:tensorflow.bzl", "if_not_windows")
 
 licenses(["notice"])  # Apache 2
 
 package(default_visibility = ["//visibility:public"])
 
-sh_binary(
+py_binary(
     name = "install_xla_in_source_tree",
-    srcs = ["install_xla_in_source_tree.sh"],
+    srcs = ["install_xla_in_source_tree.py"],
```

Comment:
It looks like you ported this to Python? Did you omit the `install_xla_in_source_tree.py` script from the PR?